lib/dynamic_config/dcdo.rb

```diff
@@ -52,6 +52,7 @@ def frontend_config
       cpa_experience: DCDO.get('cpa_experience', false),
       gender: DCDO.get('gender', false),
       'show-coteacher-ui': DCDO.get('show-coteacher-ui', false),
+      'amplitude-event-sample-rates': DCDO.get('amplitude-event-sample-rates', false)
```

Comment:
should the default be an empty object?

Reply:
💯 I changed it to return an empty object.